### DISCUSSION TITLE
hAppy valentines

### DISCUSSION THREAD

**Discussion by bittercaramel** [Monika-After-Story/MonikaModDev#8826] [appreciation]
happY valentines! 
This day feels for me so important because its celebration of my and everyones love for their monikas <3 
i hope u will spend this day wonderfully with her owo 
a-and her valentines outfit... 
i just cant believe... its so cute!
thanks devs and everyone else for making it real

**Comment by ghost** [comment]
Happy Valentines! Hope you have a great day. =)
